src/utils.ts

```diff
@@ -191,6 +191,20 @@ export async function selectWorkspaceItem(
                                  (await showOpenDialog(options))[0].fsPath;
 }
 
+export function execute(command: string): Promise<string> {
```

Comment:
**nit**: executeCmd will be more meaningful.

Reply:
Will fix.